oss-internship-2020/curl/examples/example1.cc

```diff
@@ -12,67 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <libgen.h>
-#include <syscall.h>
 #include <iostream>
+
 #include "curl_sapi.sapi.h"
 #include "sandboxed_api/util/flag.h"
 
-/*
-int main() {
-    CURL *curl = curl_easy_init();
-    curl_easy_setopt(curl, CURLOPT_VERBOSE, 1l);
-    curl_easy_setopt(curl, CURLOPT_URL, "http://example.com");
-    curl_easy_perform(curl);
-    curl_easy_cleanup(curl);
-}
-*/
-
 class CurlApiSandboxEx1 : public CurlSandbox {
-
-  std::unique_ptr<sandbox2::Policy> ModifyPolicy(
-      sandbox2::PolicyBuilder*) override {
-
-    return sandbox2::PolicyBuilder()
-        .DangerDefaultAllowAll()
-        .AllowUnrestrictedNetworking()
-        .AddDirectory("/lib")
-        .BuildOrDie();
-  }
-
+  private:
```

Comment:
It'd be nice if you would refer to the example program from the Curl documentation by adding a comment at the top.